ApplicationCode/ProjectDataModel/RimGeoMechModels.cpp

```diff
@@ -29,14 +36,84 @@ RimGeoMechModels::RimGeoMechModels( void )
 {
     CAF_PDM_InitObject( "Geomechanical Models", ":/GeoMechCases48x48.png", "", "" );
 
-    CAF_PDM_InitFieldNoDefault( &cases, "Cases", "", "", "", "" );
-    cases.uiCapability()->setUiHidden( true );
+    CAF_PDM_InitFieldNoDefault( &m_cases, "Cases", "", "", "", "" );
+    m_cases.uiCapability()->setUiHidden( true );
 }
 
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
 RimGeoMechModels::~RimGeoMechModels( void )
 {
-    cases.deleteAllChildObjects();
+    m_cases.deleteAllChildObjects();
```

Comment:
This is not necessary. The destructor of a ChildArrayField will delete all contained objects